flash.py

```
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+import os
+import subprocess
+import re
+
+def downloadESPtool():
+    output = subprocess.run(["sudo", "apt", "install", "esptool"], text=True)
+    if(output.returncode != 0):
+        raise Exeption(output.returncode)
+        
+def getNumber(maxInput):
+    num = 0
+    while True:
+        try:
+            num = int(input())
+            if (num >= 0 and num < maxInput):
+                break;
+            else: 
+                raise Exception("Wrong input number")
+        except:
```

Don't use bare except. What if I want to exit at this location with ctrl-c?